src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java

```diff
@@ -49,8 +49,6 @@
   public static final String FILE = "Image001.jpg";
   public static final String FILEPNG = "Image001.png";
   public static final String FILE2PNG = "Image002.png";
-  public static final String SMALLFILE = "Image011-Small-image.png";
-  public static final String BROKENEXTENSIONFILE = "broken-extension-image.fer";
```

Comment:
Do we need to remove those files as well?

Reply:
done